flink-connector-aws/flink-connector-aws-kinesis-streams/src/main/java/org/apache/flink/connector/kinesis/source/enumerator/KinesisStreamsSourceEnumerator.java

```diff
@@ -30,10 +30,6 @@
 import org.apache.flink.connector.kinesis.source.split.KinesisShardSplit;
 import org.apache.flink.connector.kinesis.source.split.StartingPosition;
 
-import org.apache.flink.shaded.guava30.com.google.common.collect.ImmutableList;
```

Comment:
Are we getting rid of guava ? 
I dont see any dependency removal of guava in pom file ?

Reply:
This comes from the shaded guava! We are removing this dependency so that we don't have to keep upgrading this when Flink upgrades shaded guava

Reply:
All the changes that changes `ImmutableXXX` to `Streams` were in tests. We do not need to use a `ImmutableXXX`